tasks/Prisoner's Dilemma (various versions).md

```diff
@@ -0,0 +1,66 @@
+The “Give Some” Game
+- Participants are broken up into groups of five.
+- Players are briefed on the rules.
+- Each player decides whether to keep $8.00 from the experimenter, or give $3.00 to every other participant in the group.
+- This is repeated for as many rounds as the experimenter chooses.
+General Prisonniers Dilemma
+- Generally played with two people, and can be played repeatedly.
+- Designed to model a scenario where where two prisoners were caught for jointly committing a felony.
+- The two prisoners are held in separate rooms so that they cannot communicate.  The District Attorney cannot prove their guilt but offers each prisoner a plea deal.  If one confesses, they get to go free, but the other receives the maximum sentence.  If they both confess they receive a moderate sentence each.  If neither confess, both receive a minimum sentence.
+
+## Roles 
+Participants work independently within their groups, they can choose to be cooperative or act selfishly.
+
+## Instructions
+Instructions developed by CSS lab  
+
+The “Take Some” Game
+- In this game you and two other participants will work in a group.  
+- Each round you will have a choice of holding up either the red or blue poker chip.  
+- The red chip will award you $3.00, but will deducted $1.00 from each participant, including yourself.
+- The blue chip will award you $1.00 and will not deduct anything from any participant.
+- It is plain to see the benefits of sharing.
+- Each participant will hold up their choice at the same time, and you will not know what the others are selecting before the round.
+- The experiment will run for N rounds, and at the end your total amount earned will be tallied. 
+
+The “Give Some” Game
+- In this experiment you will work in a group with four other participants.
+- In each round of the experiment you and every other participant will make an independent decision.  The decision will be your choice of two options. The options are as follows:
+    - You can decide to take an $8.00 payout from the experimenter.
+    - Or you can decide to give every other participant a $3.00 gift.
+- It is plain to see the benefits of gifting if everyone is cooperative.
+- The experiment will run for N rounds, and at the end your total amount earned will be tallied.
+
+For more specific information on instructions, check these links to other forms of the prisoners dilemma:  
+Intergroup Prisoner's Dilemma: https://github.com/Watts-Lab/task-mapping/blob/61797c06d83b8c49dff192a7e909077ade004edd/tasks/Intergroup%20Prisoner's%20Dilemma.md  
+Intergroup Prisoner's Dilemma-Maximizing Difference: https://github.com/Watts-Lab/task-mapping/blob/61797c06d83b8c49dff192a7e909077ade004edd/tasks/Intergroup%20Prisoner's%20Dilemma-Maximizing%20Difference.md
+
+# Criteria
+## Performance calculation
+The performance of a group can be measured by their payout, i.e. how well they cooperate, and their ability to identify the most beneficial strategy.
+
```

Comment:
I will probably add some information from Table 1 in the paper such as what payout is depending on:
1. number of participants 
2. number of cooperators/defectors